synapse/api/ratelimiting.py

```diff
@@ -76,6 +77,7 @@ async def can_do_action(
             burst_count: How many actions that can be performed before being limited.
                 Overrides the value set during instantiation if set.
             update: Whether to count this check as performing the action
+            nb_actions: The number of actions the user wants to do.
```

Comment:
by the way I would just call this `n_actions`. `nb` isn't really an abbreviation I'm familiar with. It's not important though.